src/java.base/share/classes/java/time/LocalDateTime.java

```diff
@@ -2019,4 +2022,11 @@ static LocalDateTime readExternal(DataInput in) throws IOException {
         return LocalDateTime.of(date, time);
     }
 
+    static {
+        SharedSecrets.setJavaTimeAccess(new JavaTimeAccess() {
+            public void formatTo(StringBuilder buf, LocalDateTime ldt) {
+                ldt.formatTo(buf);
```

Comment:
Neither java.time nor the rest of the JDK has a convention to use acronyms like `ldt` as variable names.

Maybe this variable should be called for example `timeToFormat` instead?